### PR TITLE
Add support for Product, ProductSet

### PR DESCRIPTION
**Pull request by shawntian-ai**

### CODE REVIEW

**Comment by shawntian-ai** [comment]
@suzaku这个是暂时不会上线到 master 的代码，不

File: examples/src/cn/productai/api/examples/Main.java [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
+import cn.productai.api.examples.service.ServiceApiExample;
 
 public class Main {
 
     public static void main(String[] args) {
 
         IProfile profile = new DefaultProfile();
-        profile.setAccessKeyId("****");
-        profile.setSecretKey("****");
+        profile.setAccessKeyId("249f5ce644aae36b6eb138e87af9e2fd");
```
Comment (evitself) [inline examples/src/cn/productai/api/examples/Main.java]: @shawntian-ai please remove your own key pair

File: src/main/java/cn/productai/api/pai/base/ProductBatchAddByFileBaseRequest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        this.csvFile = csvFile;
+    }
+
+    public ProductBatchAddByFileBaseRequest() {
+        super();
+    }
+
+    @Override
+    public String getApiUrl() {
+        return String.format("https://%s/product_sets/_0000178/%s/products", this.getHost(), this.getProductSetId());
```
Comment (evitself) [inline src/main/java/cn/productai/api/pai/base/ProductBatchAddByFileBaseRequest.java]: @shawntian-ai just wondering if we can set this string as const? Things like _0000178 looks to be some kind of magic number, and there are 10+ occurrences across the files.

File: examples/src/cn/productai/api/examples/Main.java (hunk 1 above)
Comment (shawntian-ai) [reply examples/src/cn/productai/api/examples/Main.java]: fixed

File: src/main/java/cn/productai/api/pai/base/ProductBatchAddByFileBaseRequest.java (hunk 2 above)
Comment (shawntian-ai) [reply src/main/java/cn/productai/api/pai/base/ProductBatchAddByFileBaseRequest.java]: fixed

**Comment by shawntian-ai** [comment]
@evitself all faults have been fixed

**Pull request merged**